client-rpc/src/rpc/transaction_rpc.rs

```diff
@@ -0,0 +1,168 @@
+use jsonrpc_core::Result;
+use jsonrpc_derive::rpc;
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+use chain_core::tx::data::access::{TxAccess, TxAccessPolicy};
+use chain_core::tx::data::attribute::TxAttributes;
+use chain_core::tx::data::input::TxoPointer;
+use chain_core::tx::data::output::TxOut;
+use chain_core::tx::data::{Tx, TxId};
+use chain_core::tx::TransactionId;
+use client_common::{PublicKey, Result as CommonResult};
+
+use crate::server::to_rpc_error;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct RawTransaction {
+    tx: Tx,
+    tx_id: TxId,
+}
+
+#[rpc]
+pub trait TransactionRpc: Send + Sync {
+    #[rpc(name = "transaction_createRaw")]
+    fn create_raw(
+        &self,
+        inputs: Vec<TxoPointer>,
+        outputs: Vec<TxOut>,
+        view_keys: Vec<String>,
```

Comment:
isn't there a serde-implementation: https://github.com/crypto-com/rust-secp256k1-zkp/blob/master/src/key.rs#L370 ? 
but I guess that one would be an array of numbers instead of some string?

Reply:
I have to experiment it and will give the experiment results here.

Reply:
I tried and it will require an array to deserialize and serialize as array of numbers as well. I have implemented custom de/serializer here.